javascript-modal/styles.css

```diff
@@ -0,0 +1,57 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  height: 900px;
+  background-color: white;
+}
+
+.modal-button {
+  background-color: rgb(74, 74, 233);
+  color: white;
+  padding: 15px 40px 15px 40px;
+  font-size: 25px;
+  cursor: pointer;
+  border: none;
+  border-radius: 5px;
+}
+
+.no-button {
+  background-color: rgb(235, 93, 93);
+  color: white;
+  font-size: 20px;
+  padding: 8px 25px 8px 25px;
+  cursor: pointer;
+  border: none;
+  border-radius: 5px;
+}
+
+.no-button-align {
+  text-align: center;
+}
+
+h2 {
+  text-align: center;
+}
+
+.modal-window {
+  position: fixed;
```

Comment:
Rather than making the modal window itself `position: fixed`, I would make the overlay the element that is `position: fixed`.